NebulaPatcher/Patches/Dynamic/BuildTool_Path_Patch.cs

```diff
@@ -0,0 +1,45 @@
+﻿using HarmonyLib;
+using NebulaWorld;
+using NebulaWorld.Factory;
+using UnityEngine;
+
+namespace NebulaPatcher.Patches.Dynamic
+{
+    [HarmonyPatch(typeof(BuildTool_Path))]
+    class BuildTool_Path_Patch
+    {
+        [HarmonyPrefix]
+        [HarmonyPatch("CreatePrebuilds")]
+        public static bool CreatePrebuilds_Prefix(BuildTool_Path __instance)
+        {
+            return BuildToolManager.CreatePrebuilds(__instance);
+        }
+
+        [HarmonyPostfix]
+        [HarmonyPatch("CreatePrebuilds")]
+        public static void CreatePrebuilds_Postfix()
+        {
+            if(SimulatedWorld.Initialized && !LocalPlayer.IsMasterClient && FactoryManager.EventFromServer && FactoryManager.IsHumanInput)
```

Comment:
Please explain what this is intended to do so people don't have to git blame to find out